new-components/enrichers/custom-annotation/internal/annotation/annotation_test.go

```diff
@@ -0,0 +1,253 @@
+package annotation_test
+
+import (
+	"context"
+	"encoding/json"
+	"strings"
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	"google.golang.org/protobuf/types/known/structpb"
+
+	ocsf "github.com/smithy-security/smithy/sdk/gen/com/github/ocsf/ocsf_schema/v1"
+
+	"github.com/smithy-security/smithy/new-components/enrichers/custom-annotation/internal/annotation"
+)
+
+func ptr[T any](v T) *T {
```

Comment:
I created that function for gosec only since it manipulates the actual `ocsf.VulnerabilityFinding` extensively using pointers (sad face here btw, very bad to use) - here I only needed it for tests

Reply:
how about we add this to `github.com/smithy-security/smithy/utils`?